memetotext.py

```diff
@@ -0,0 +1,10 @@
+from PIL import Image
+import pytesseract
+from TamilSpellingAutoCorrect import TamilSpellingAutoCorrect
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe' 
```

Comment:
I guess we must avoid something like this. this will become a problem when we run our code in different environments